test-e2e/sync.js

```diff
@@ -26,8 +26,11 @@ import { kSyncState } from '../src/sync/sync-api.js'
 
 const SCHEMAS_INITIAL_SYNC = ['preset', 'field']
 
-test('Create and sync data', { timeout: 100_000 }, async function (t) {
-  const COUNT = 10
+test('Create and sync data', { timeout: 60_000 }, async (t) => {
+  // NOTE: Unlike other tests in this file, this test uses `node:assert` instead
+  // of `t` to ease our transition away from Brittle. We can remove this comment
+  // when Brittle is removed.
+  const COUNT = 5
```

Comment:
having this stay higher to stress test seems generally useful, but maybe there's an alternative approach to achieve that? i know we introduced something like a `FAST_TESTS` env variable but i don't think it's being used 🤔

Reply:
Nah, I think it's useful too. Reverted to 10 in 340bf820ebf500b4f7a3bdb0964e575ff8053158. I might consider using `FAST_TESTS` in the future, but not necessary now.

Reply:
curious how much faster the test is now for you, since the only thing related change is parallelizing the assertions?

Reply:
It sped things up by about 10 seconds on my machine.